app/helpers/application_helper.rb

```diff
@@ -20,7 +20,7 @@ def add_rtl_body_class(other_classes)
 
   def favicon_path
     env_suffix = Rails.env.production? ? '' : '-dev'
-    asset_path "favicon#{env_suffix}.ico"
+    asset_path "favicon#{env_suffix}.ico", skip_pipeline: true
```

Comment:
Since this will always just be in `public` maybe we don't even need to use asset_path here?